src/releasers/ruby.ts

```diff
@@ -41,7 +41,9 @@ export class Ruby extends ReleasePR {
   }
   protected async _run(): Promise<number | undefined> {
     const latestTag: GitHubTag | undefined = await this.gh.latestTag(
-      this.monorepoTags ? `${this.packageName}-` : undefined
+      this.monorepoTags ? `${this.packageName}/` : undefined,
+      false,
+      this.monorepoTags ? `${this.packageName}` : undefined
```

Comment:
This, I think, could still end with a dash.

Reply:
This surfaced a bug, I think we should allow someone to provide a `-` suffix, but the logic was assuming someone did not.

I've added a test.